src/puppetlabs/jdbc_util/core.clj

```diff
@@ -156,3 +156,31 @@
                        [sql 0]
                        seq-params-w-indices)]
     (vec (conj (flatten parameters) sql'))))
+
+(defn has-extension [db extension]
+  (-> (jdbc/query db ["select count(*) from pg_extension where extname = ?" extension])
+      first
+      :count
+      pos?))
+
+(defn has-pglogical-extension [db]
+  (has-extension db "pglogical"))
+
+(defn- unsafe-escape-sql-string
+  "Escape the given string so it can be \"safely\" passed as a string parameter
+  to an sql query."
+  [s]
+  (clojure.string/replace s "'" "''"))
+
+(defn wrap-ddl-for-pglogical
+  "Wrap the given sql (presumably DDL) in a call to
+  pglogical.replicate_ddl_command, escaping quotes and wrapping the statement so
+  it won't return anything. "
+  [sql]
+  (str "do 'begin perform "
+       (unsafe-escape-sql-string
+        (str "pglogical.replicate_ddl_command('"
+             "set local search_path to public; "
```

Comment:
Inside of pglogical.replicate_ddl_command, search_path seems to be empty by default. Qualified table names work correctly, but we have a pile of migrations with unqualified names in them.